README.md

```diff
@@ -196,50 +203,15 @@ This query to get the asset materializations metadata and asset dependencies (li
 
 ## Source datasets
 
-### BAG
```

Comment:
I transferred all this in another md file because the readme was getting to long. Also I updated the info about BAG and I am going to add  info for the other datasets too.